src/modules/app/config/edc-api.service.ts

```diff
@@ -0,0 +1,28 @@
+import {Injectable} from '@angular/core';
```

Comment:
Is this the correct place to store this file? It is not a direct application configuration or is it? It is constructor with defined endpoints => why not put it in app/api folder or something.